glossary/README.md

```diff
@@ -0,0 +1,3 @@
+# Glossary
```

Comment:
```suggestion
# Cosmos Blockchain Glossary
```

Reply:
to improve our SEO, let's include Cosmos and Blockchain in the title of the glossary. (Is there a way to improve our findability? more metadata perhaps?)

Reply:
Hmm - to improve our discoverability, we'll need to tweak the meta title and description of `cosmos.network/glossary`. Once that's done, we'll add `cosmos.network/glossary` to [the sitemap](http://cosmos.network/sitemap.xml). So Google can tell where to crawl the URLs for the specific site.

Reply:
meta title and description tweaking, I'm up for that! @lovincyrus how can we tweak?